### ISSUE TITLE
quickinstall.py and "m" need to die

### ISSUE THREAD

**Issue by ThomasWaldmann** [moinwiki/moin#1621]
These are nice convenience wrappers for developers and "git checkout users", but for a normal installation (like after pip install or after installing a moin linux dist package) I guess they won't work.

Also, these are VERY special to moin and feel alien even to users familiar with the usual python installation / usage workflows.

These 2 scripts do all sorts of stuff and depending on the target user group I think we need different ways to replace these scripts:
- for the developer-only oriented helpers, I guess we could use `tox` or even `make`. For another project I just created a `scripts/make.py` to do misc. stuff there (including a `clean` subcommand, replacing `python setup.py clean`). No stuff that a end users needs should be in there.
- for the moin admins / "users", I think we either should integrate into a `moin` (or `moin-admin`) script command (these will be provided by `pip install` and also by a linux dist package install).
- guess everything dealing with virtual envs should be removed from our helpers. That is outside of the scope that moin tools should cover and there are good tools for that, like e.g. `pipx`, or just the basic `python -m venv` python users are very familiar with. Pycharm, for example, also supports devs creating and activating venvs and trying to do that again in moin might rather collide with such stuff.

**Comment by ReimarBauer** [comment]
When there is a pypi package we can use [grayskull](https://github.com/conda/grayskull) to create a conda-forge package. I also added there some of the xstatic packages already.

**Comment by sfermigier** [comment]
Yes. I think once the virtualenv is set up (and there are tools for that: poetry, pdm/hatch, rye...), administrative commands should be available via subcommands of the `flask` CLI.

**Comment by ThomasWaldmann** [comment]
Guess this should be fixed before release?

`m` and `quickinstall` were hacks addressed to developers, but they shouldn't be required for users - moin should behave and be usable like a normal python project.
